bin/yaml/crystal.yaml

```diff
@@ -15,9 +15,9 @@ compilers:
       - ln -s /opt/compiler-explorer/libs/libpcre/8.45/x86_64/lib/libpcre.a # for 1.1.1 and before
       - ln -s /opt/compiler-explorer/libs/libevent/2.1.12/x86_64/lib/libevent.a
     targets:
-      - 1.3.1
```

Comment:
Please can you leave the old targets in? thanks!

Reply:
Argh I'm sorry! @RubenRBS corrects me here, we're doing aliasing. Please disregard :blush: